subs:

    left_img:

      topic: /darknet_ros/left/image_color
      queue_size: 1

    depth_img:

      topic: /darknet_ros/depth/image_raw
      queue_size: 1

    left_info:

      topic: /mynteye/left/camera_info
      queue_size: 1

    depth_info:

      topic: /mynteye/depth/camera_info
      queue_size: 1

    odom:

      topic: /mynteye/odom
      queue_size: 1  

    boxes:

      topic: /darknet_ros/bounding_boxes 
      queue_size: 1
	  num_obj_topic: /darknet_ros/object_detector

pubs:

  objects_position:

    topic: /object_poses
    queue_size: 1

using_YOLO: true
